docs/simulators/OpenFOAM.md

```diff
@@ -126,6 +126,323 @@ task.download_outputs()
 machine_group.terminate()
 ````
 
+## Advanced Example: OpenFOAM Simulation (ExaFOAM Benchmark)
+
+Here's a guide for running an OpenFOAM simulation using the Inductiva platform,
+focusing on a high-lift configuration case from the `rhoPimpleFoam` solver.
+This specific example is based on one of the benchmark cases provided by the
+[ExaFOAM project](https://exafoam.eu), which aims to develop highly
+scalable CFD solvers using OpenFOAM for exascale computing. We'll use a powerful
+cloud setup to handle the computationally intensive mesh generation and simulation.
+
+We will run a complex OpenFOAM simulation based on a high-lift configuration
+setup, which is part of the [ExaFOAM benchmarks](https://exafoam.eu/benchmarks/).
+Specifically, this case corresponds to the MB9 micro-benchmark, which is
+preparatory work leading up to the HPC Grand Challenge test case of the High
+Lift Common Research Model (CRM-HL). The CRM-HL is a full aircraft configuration
+with deployed high-lift devices, simulated using wall-modeled LES (WMLES).
+
+The MB9 micro-benchmark captures the essential characteristics of the Grand
+Challenge (such as flow physics and simulation approach) while requiring
+significantly fewer computational resources. It features a two-dimensional,
+three-element high-lift wing configuration, simulated with the IDDES model,
+which provides WMLES functionality in regions of resolved near-wall turbulence.
+This case is based on the well-known 30P30N test case, extensively studied in
+the 4th AIAA CFD High Lift Prediction Workshop (HLPW-4) and supported by
+available experimental data.
+
```

Comment:
In this example, we will run a modified version of this benchmark using two hardware configurations powered by Inductiva. First, we will run the simulation workflow on a large 360 vCPU machine. This is one of the largest single node configuration one can use to run simulations at Inductiva. Then, to see if we can halve the execution time of the simulation, we will run the simulation on a MPI cluster with 2 of these machines large machines.